internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -273,6 +278,11 @@ func (k *KafkaManager) calculateCapacityByRegionAndInstanceTypeForManualClusters
 				availableByInstanceType = 0
 			}
 
+			if cluster.ClusterType == api.EnterpriseDataPlaneClusterType {
+				maxAvailable = float64(cluster.KafkaInstanceLimit)
+				availableByInstanceType = maxAvailable - float64(totalUsed)
+			}
```

Comment:
enterprise clusters will never be part of the manual list. They'll co-exists with clusters added in the manual list but they'll only be in the data base. This check being englobed by https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/11d2609bfca4435ded886e1899c68e438a96bae6/internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go#L247 will mean that if we switch to manual scaling and we've enterprise clusters, then we'll never see metrics for enterprise clusters. 

I suggest we treat enterprise clusters outside the loop in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/11d2609bfca4435ded886e1899c68e438a96bae6/internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go#L247 

It'll be another loop that goes through `streamingUnitsByRegion` and finds only enterprise clusters, treats them and appending them onto the `result` list.